static/app/components/feedback/useFetchFeedbackData.tsx

```diff
@@ -52,7 +52,7 @@ export default function useFetchFeedbackData({feedbackId}: Props) {
     if (issueResult.isFetched && issueData && !issueData.hasSeen) {
       markAsRead(true);
     }
-  }, [issueResult.isFetched]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [issueData, issueResult.isFetched, markAsRead]);
```

Comment:
This seemed wrong before. Because we were just looking at `isFetched` i think if you toggled back and forth between two unread feedbacks (so both are in cache), then we wouldn't re-run the useEffect when the visible feedback changed. We want to re-run it in case something else has flipped the `hasSeen` bit and we want to reset it.